tests/pipeline/test_numerical_expression.py

```diff
@@ -148,39 +150,40 @@ def test_validate_bad(self):
             (f > f) > f
 
     def test_many_inputs(self):
-        factors = []
-
-        # Create 11 factors to use as inputs. We need at least 11 to create a
-        # string expression containing an x_10. Then, ensure that this long
-        # expression can correctly merge with another.
-        for class_name in 'PQRSTUVWXYZ':
-            factors.append(
-                type(
-                    class_name,
-                    (Factor,),
-                    dict(dtype=float64_dtype, inputs=(), window_length=0),
-                )
+        """
+        Test adding NumericalExpressions with >10 inputs.
+        """
+        # Create an initial NumericalExpression by adding two factors together.
+        f = self.f
+        expr = f + f
+
+        new_factor_names = ascii_uppercase.replace('F', '')
```

Comment:
Is this actually necessary?  I don't think it's a big deal to have two factors named `F` in the expression.